src/display/touch_manager.js

```diff
@@ -169,13 +175,13 @@ class TouchManager {
   #onTouchEnd(evt) {
     this.#touchMoveAC.abort();
     this.#touchMoveAC = null;
+    this.#onPinchEnd?.();
 
     if (!this.#touchInfo) {
       return;
     }
 
     if (this.#isPinching) {
-      this.#onPinchEnd?.();
       this.#isPinching = false;
     }
```

Comment:
Can't we just *unconditionally* set `this.#isPinching = false;` now and not bother with the `if`-block any more?